doc/nrf/templates/cheat_sheet.rst

```diff
@@ -186,7 +186,7 @@ Read out an objects.inv file::
 Doxygen
 =======
 
-.. doxygenfunction:: event_manager_init
+.. doxygenfunction:: profiler_init
```

Comment:
@ru-fu btw. can we create some special dummy function for cheat_sheet? Otherwise we will get duplicate declarations warning when we are documenting function which is used here